### PR TITLE
Remove default value for sku_name variable.

### PR DESCRIPTION
**Pull request by mokainos**
Resolves # . (This is applicable only if this pull request relates to a GitHub issue, delete the line otherwise)

Notes:
*
*
*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ This repository contains the module that enables you to create a Redis PaaS inst
 
 ## Usage
 
+### Recommended example for cost optimisation
+
+Premium redis PaaS instance are very expensive, care **must** be taken when using them and only used in required environments.
```
Comment (timja) [inline README.md:9]: is there a nagger in place?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,40 @@ This repository contains the module that enables you to create a Redis PaaS inst
...
+  description = "The SKU family/pricing group to use. Valid values are `C` (for Basic/Standard SKU family) and `P` (for Premium). Use P for higher availability, but beware it costs a lot more."
+}
+
+variable "sku_name" {
+  description = "The SKU of Redis to use. Possible values are `Basic`, `Standard` and `Premium`."
+}
+
+variable "capacity" {
+  description = "The size of the Redis cache to deploy. Valid values are 1, 2, 3, 4, 5"
+}
```
Comment (timja) [inline README.md:25]: these should all have defaults set for non production

File: README.md [hunk 3]
```diff
@@ -4,6 +4,40 @@ This repository contains the module that enables you to create a Redis PaaS inst
...
+```
+
+prod.tfvars
+```
+sku_name = "Premium"
+family   = "P"
+capacity = "1"
+```
+
+aat.tfvars
```
Comment (timja) [inline README.md:35]: probably not needed I would just add prod

File: variables.tf [hunk 4]
```diff
@@ -37,17 +37,17 @@ variable "minimum_tls_version" {
 }
 
 variable "family" {
-  default     = "P"
+  default     = "C"
```
Comment (timja) [inline variables.tf:40]: 🚫 This is a breaking change and can't be done without a nagger in place first and even then I think you would just want to remove the default

File: variables.tf [hunk 5]
```diff
@@ -37,17 +37,17 @@ variable "minimum_tls_version" {
...
   description = "The SKU family/pricing group to use. Valid values are `C` (for Basic/Standard SKU family) and `P` (for Premium). Use P for higher availability, but beware it costs a lot more."
 }
 
 variable "sku_name" {
-  default     = "Premium"
+  default     = "Basic"
```
Comment (timja) [inline variables.tf:45]: 🚫 This is a breaking change and can't be done without a nagger in place first and even then I think you would just want to remove the default

File: variables.tf [hunk 6]
```diff
@@ -37,17 +37,17 @@ variable "minimum_tls_version" {
...
   description = "The SKU of Redis to use. Possible values are `Basic`, `Standard` and `Premium`."
 }
 
 variable "capacity" {
-  default     = "1"
+  default     = "3"
```
Comment (timja) [inline variables.tf:50]: Why change this to 3?

**Review by timja — CHANGES_REQUESTED** [review]

**Comment by madjava** [comment]
Closing PR for now

**Pull request closed without merging**